src/Indicator.vala

```diff
@@ -361,13 +422,57 @@ public class Sound.Indicator : Wingpanel.Indicator {
         }
     }
 
+    uint blubble_timeout_id = 0;
     private void play_sound_blubble () {
-        Canberra.Proplist props;
-        Canberra.Proplist.create (out props);
-        props.sets (Canberra.PROP_CANBERRA_CACHE_CONTROL, "volatile");
-        props.sets (Canberra.PROP_EVENT_ID, "audio-volume-change");
-        ca_context.play_full (0, props);
+        if (blubble_timeout_id > 0) {
+            return;
+        }
+
+        blubble_timeout_id = Timeout.add (50, () => {
+            if (!show_notification ()) { /* false when indicator open or notification cannot be shown */
```

Comment:
I think this logic is a bit unclear, I'll try to do a PR against your Pr.

Reply:
You may be right.  I was probably trying to DRY the code a bit too much.

Reply:
Suggested changes incorporated